django-stubs/core/cache/backends/base.pyi

```diff
@@ -24,7 +24,7 @@ class BaseCache:
     def set(self, key: Any, value: Any, timeout: Any = ..., version: Optional[Any] = ...) -> None: ...
     def touch(self, key: Any, timeout: Any = ..., version: Optional[Any] = ...) -> bool: ...
     def delete(self, key: Any, version: Optional[Any] = ...) -> None: ...
-    def get_many(self, keys: List[str], version: Optional[int] = ...) -> Dict[str, Union[int, str]]: ...
+    def get_many(self, keys: List[str], version: Optional[int] = ...) -> Dict[str, Any]: ...
```

Comment:
Why we have `keys` as `List[str]` and not `List[Any]`?

Reply:
And is it a `List`? Or `Sequence`?

Reply:
Seems it depend on the SubClass implementation of BaseCache. As in Django implementation, I think it can be Sequence, and keys child can be of type Any.

Reply:
Let's change it then!

Reply:
Sure, adding new changes

Reply:
should I change for other functions as well?

Reply:
Please change them, if they have the same problem.

Reply:
Done